packages/e2e/src/specs/contacts/new-contact.e2e.ts

```diff
@@ -3,123 +3,302 @@
-    })
-    const deleteOption = await ContactsPage.deleteContactOptionMenu
+    await contactsPage.optionsButtonOnContactListClick()
+    await browser.pause(2000)
+    const deleteOption = await contactsPage.deleteContactOptionMenu
     await deleteOption.click()
 
-    ModalContactsPage.buttonConfirmDeleteClick()
-    await browser.executeAsync((done) => {
-      setTimeout(done, 3000)
-    })
+    modalContactsPage.buttonConfirmDeleteClick()
+    await browser.pause(3000)
 
-    const noContactsTextLabel = await ContactsPage.noContactsTextLabel
+    const noContactsTextLabel = await contactsPage.noContactsTextLabel
 
     await noContactsTextLabel.waitForDisplayed()
-    const noContactsH3Value = await noContactsTextLabel.$("<h3>")
-    const noContactsPValue = await noContactsTextLabel.$("<p>")
-    await expect(noContactsH3Value).toHaveText("No contacts found")
-    await expect(noContactsPValue).toHaveText(
-      "Search results do not match any contact."
+    await expect(noContactsTextLabel).toBeDisplayed()
+    await expect(noContactsTextLabel).toHaveTextContaining([
+      "You don't have any contacts yet",
+      "Press New Contact to add one.",
+    ])
+  })
+})
+describe("Favourites contact scenarios", () => {
+  const contactToAdd = contacts[0]
+  const contactToEdit = contacts[2]
+
+  before(async () => {
+    // Waiting for device connected through USB
+    await browser.pause(10000)
+    await modalGeneralPage.clickUpdateAvailableModalCloseButton()
+    await navigationTabs.clickContactsTab()
+  })
+
+  it("Should press new contact button, add contact data and save it", async () => {
+    const newContactButton = await contactsPage.newContactButton
+    await waitForClickableAndClick(newContactButton)
+    await expect(newContactButton).toBeDisabled()
+
+    await contactsPage.insertTextToFirstNameInput(contactToAdd.firstName)
+    await contactsPage.insertTextToLastNameInput(contactToAdd.lastName)
+    await contactsPage.insertTextToPrimaryPhoneNumberInput(
+      contactToAdd.primaryNumber
+    )
+    await contactsPage.insertTextToSecondaryPhoneNumberInput(
+      contactToAdd.secondaryNumber
+    )
+    await contactsPage.insertTextToAddressFirstLineInput(
+      contactToAdd.addressFirstLine
     )
+    await contactsPage.insertTextToAddressSecondLineInput(
+      contactToAdd.addressSecondLine
+    )
+    if (contactToAdd.addToFavourites) {
+      await contactsPage.addToFavouritessCheckbox.click()
+    }
+    await contactsPage.clickSaveContactButton()
+    await browser.pause(2000)
+    const storedContacts = await contactsPage.listOfContacts
+    expect(storedContacts.length).toBeGreaterThan(0)
+    const ContactData = await storedContacts[0].getText()
+    expect(ContactData).toHaveText(
+      `${contacts[0].firstName} ${contacts[0].lastName}`
+    )
+
+    const phoneNumber = await contactsPage.phoneNumberOnContactList
+    expect(phoneNumber).toHaveTextContaining([
+      contacts[0].primaryNumber,
```

Comment:
I think you want to use const `contactToAdd`

Reply:
https://github.com/mudita/mudita-center/pull/1201/commits/08252851ec674568c6c4262bcebf0327ead77165